### PR TITLE
chore: Add loveramp mock data and .env file

### PR DESCRIPTION
**Pull request by ailsoncgt**
Similar to [PR 110](https://github.com/fiatconnect/validate/pull/110)

- Add small image to base64 validation
- Update env value for QUOTE_OUT_MOCK

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -4,3 +4,5 @@ export const ALFAJORES_CHAIN_ID = 44787
 export const WEBHOOK_RECIPIENT_BASE_URL =
   'https://liquidity-dot-celo-mobile-alfajores.appspot.com'
 export const WEBHOOK_RECIPIENT_HISTORY_ENDPOINT = '/fiatconnect/webhook/history'
+export const BASE64_IMAGE =
```
Comment (cajubelt) [inline src/constants.ts:7]: this is not a valid base64 image

Comment (cajubelt) [reply src/constants.ts:7]: it has symbols not included in the [base64 alphabet](https://en.wikipedia.org/wiki/Base64) and failed to decode when I tried to put it into an online base64 image decoder

Comment (cajubelt) [reply src/constants.ts:7]: if you just need some base64 image you could copy an illustration of an ID card, which would be smaller than a "real" picture and not contain any PII

File: src/mock-data/quote.ts [hunk 2]
```diff
@@ -37,4 +44,5 @@ export const MOCK_QUOTE: Record<string, Omit<QuoteRequestBody, 'address'>> = {
   quoteOutNigeriaCUSD,
   quoteOutXOFCUSD,
   quoteInXOFCUSD,
+  quoteOutBrazilCREAL: quoteBrazilCREAL,
```
Comment (cajubelt) [inline src/mock-data/quote.ts:47]:
```suggestion
  quoteBrazilCREAL,
```
not important but a bit nicer

Comment (ailsoncgt) [reply src/mock-data/quote.ts:47]: ok, but will need to change quote_out_mock on env file.

Comment (cajubelt) [reply src/mock-data/quote.ts]: oh whoops I didn't read closely and didn't see the name difference

Comment (cajubelt) [reply src/mock-data/quote.ts]: lol

**Review by cajubelt — APPROVED** [review]

**Pull request merged**